src/nodedef.cpp

```diff
@@ -168,23 +168,45 @@ void NodeBox::deSerialize(std::istream &is)
 	TileDef
 */
 
+#define TILE_FLAG_BACKFACE_CULLING 0x0001
+#define TILE_FLAG_TILEABLE_HORIZONTAL 0x0002
+#define TILE_FLAG_TILEABLE_VERTICAL 0x0004
+#define TILE_FLAG_HAS_COLOR 0x0008
+#define TILE_FLAG_HAS_SCALE 0x0010
+#define TILE_FLAG_HAS_ALIGN_STYLE 0x0020
+
 void TileDef::serialize(std::ostream &os, u16 protocol_version) const
 {
-	// protocol_version >= 36
-	u8 version = 5;
+	// protocol_version >= 37
+	u8 version = 6;
```

Comment:
Either support backwards compatibility in both functions (serialize/deserialize) or drop it, as we're in 0.5.0-dev anyway.